### DISCUSSION TITLE
Background Apps and Process List Updated on Amazon Appstore!

### DISCUSSION THREAD

**Discussion by visnkmr** [visnkmr/backgroundappslist#10] [General]
Hey, 
Amazon Appstore says the update for Background Apps and Process List is now live. Their update release schedule may vary based on region.

⚠️Thanks to all users who brought this to our attention.

**Comment by Sacohen9665** [comment]
I’m having the same issue I had about a month ago that say the my device time is incorrect.
![9BCF679C-E79C-44A6-9978-750C1389732B](https://user-images.githubusercontent.com/123300392/222124562-310564d3-5940-4a28-aede-4490b038c08f.jpeg)

**Reply by m22tanker** [reply]
I'm getting the same error as Sacohen9665. So what is the fix??

**Reply by Sacohen9665** [reply]
There is no fix that we can do.
We have to wait for the developer to see these messages, fix the app and re-upload it to the Amazon Store.
This happened a month or so ago and that’s what we had to do then.
And I expect this to continue to happen in the future.

**Reply by m22tanker** [reply]
Thanks for the reply. Strange, it was working fine until today March 1

**Reply by visnkmr** [reply]
If your app version number has a "G" in it, you will have to uninstall the app since its not from amazon appstore and reinstall the app from amazon appstore. The version from amazon appstore has an "A" in it. Fire TV users should always install one with "A" in version name available from amazon appstore.
1.3G7i, The Google Play store version was updated 1 week ago. No issues in the latest version.
1.3A7i, The amazon Appstore version was updated 5 weeks ago. No issues in latest version.

**Comment by Sacohen9665** [comment]
Yes, I’m guessing it had something to do with February being a short month., but last time t happened was January 23rd, if I remember correctly.

**Reply by m22tanker** [reply]
This is the first time that this has happened to me.

**Reply by visnkmr** [reply]
> Yes, I’m guessing it had something to do with February being a short month., but last time t happened was January 23rd, if I remember correctly.

Sorry, but this has nothing to do with the month. This most probably happened because our users installed app from unofficial sources last time the issue happened. If you have done so, just uninstall the installed version and install the versions available from official source as listed in our telegram channel.

**Comment by m22tanker** [comment]
Do you think trying an older version would work?

**Reply by Sacohen9665** [reply]
I don’t know of you can find an older version, but no I don’t.
If you find one and try it let me know.

**Reply by m22tanker** [reply]
Will do.

**Reply by m22tanker** [reply]
Version 1.3G7i on the following URL works. https://apkpure.com/background-apps-process-list/io.github.visnkmr.bapl

**Reply by Sacohen9665** [reply]
Thanks

**Reply by visnkmr** [reply]
Hey,
Just install the latest version available for your device from official source as listed in our telegram channel there should be no issues. This most probably happened because our users installed app from unofficial sources last time the issue happened. If you have done so, just uninstall the installed version and install the versions available from official source

**Comment by m22tanker** [comment]
You're welcome.

**Comment by visnkmr** [comment]
Hey,
If your app version number has a "G" in it, you will have to uninstall the app since its not from amazon appstore and reinstall the app from amazon appstore. The version from amazon appstore has an "A" in it. Fire TV users should always install one with "A" in version name available from amazon appstore.
The Google Play store version was updated 1 week ago.
The amazon Appstore version was updated 5 weeks ago.

**Comment by m22tanker** [comment]
Sacohen9665, Do you have the current version from the appstore? Does it work? What is the version you have?

**Reply by Sacohen9665** [reply]
Yes, I got the current version from the App Store earlier today.
It’s working as it should.
It’s version 1.302A7i.

Is this going to happen again next month because so far it happened 1/23/22 and 3/1/23.

**Reply by m22tanker** [reply]
Thanks for the reply. I hope visnkmr reads that and fixes it to keep working

**Reply by visnkmr** [reply]
> Yes, I got the current version from the App Store earlier today. It’s working as it should. It’s version 1.302A7i.
> 
> Is this going to happen again next month because so far it happened 1/23/22 and 3/1/23.

Sorry, but this has nothing to do with the month. This most probably happened because some of our users installed app version which is distributed via google play, last time the issue happened on 1/23/22. For the play store version that these users installed the issue began in 3/1/23. while the users who used the ones available from amazon appstore should have faced no issues

**Comment by ShlomobCohen** [comment]
This may be an issue with your app.  The latest release is 1.3A14i, yet the app show the update message.
Hope this helps and many thanks.

![BAPL](https://github.com/visnkmr/backgroundappslist/assets/95410397/872daa2e-b29d-4152-be68-00d0e28d8427)

**Reply by visnkmr** [reply]
Amazon appstore update should now be available as per https://github.com/visnkmr/visnkmr/issues/25#issuecomment-2083049301

**Comment by m22tanker** [comment]
Same here. visnkmr do you have an update for your app?

**Reply by visnkmr** [reply]
Hey, just got the notification of these messages today. First of all Thanks for reminding me, the update should already be available on github or our [appstore](https://visnkmr.github.io/appstore); the update has also been submitted to amazon appstore and google play which will be available after their usual review.

**Reply by visnkmr** [reply]
Amazon appstore update should now be available as per https://github.com/visnkmr/visnkmr/issues/25#issuecomment-2083049301

**Comment by jtodaro** [comment]
This seems to be out of date on the Amazon Appstore, again.

**Comment by Sou1Rider** [comment]
Vishnu, Don't forget about updating Amazon app store
<img width="1920" height="1200" alt="Screenshot_20260208-210600" src="https://github.com/user-attachments/assets/75933d19-8380-4390-a4a1-c4a4e1565d19" />
